src/ray/core_worker/pubsub/publisher.h

```diff
@@ -18,11 +18,15 @@
 #include "absl/container/flat_hash_map.h"
 #include "absl/container/flat_hash_set.h"
 #include "absl/synchronization/mutex.h"
+
+#include "ray/common/asio/instrumented_io_context.h"
 #include "ray/common/id.h"
+
 #include "src/ray/protobuf/common.pb.h"
 
 namespace ray {
 
+using SubscriberID = UniqueID;
```

Comment:
This will be part of the third PR! 

- First PR: Current stuff
- Second PR: Moving them to ray_common
- Third PR: Use templated implementation and adapt it for the core worker. 

So you are suggesting to make this ID part also a part of template right?

Reply:
Yep. That will be in the scope of the third PR!